test/unit/github_client/test_get_release.py

```diff
@@ -21,3 +22,31 @@ def test_get_latest_bundle_release_gets_latest_release(monkeypatch, input_versio
     input_version = Version.coerce(input_versions)
     latest_version, is_draft, latest_release = get_latest_bundle_release(str(input_version))
     assert str(latest_version) == output_versions
+
+
+def test_get_bundle_release_returns_none_if_no_release_found(monkeypatch):
+    mock_version_1_1 = Mock(tag_name="1.1.0")
+    mock_version_0_0 = Mock(tag_name="0.0.0.0")
```

Comment:
Is the 4 zeros intentional?
```suggestion
    mock_version_0_0 = Mock(tag_name="0.0.0")
```

Reply:
yea I want to test that it won't match if someone creates a bundle with four digits